mteb/tasks/Classification/bul/BulgarianStoreReviewSentimentClassfication.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class BulgarianStoreReviewSentimentClassfication(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="BulgarianStoreReviewSentimentClassfication",
+        description="Bulgarian online store review dataset for sentiment classification.",
+        reference="https://doi.org/10.7910/DVN/TXIK9P",
+        dataset={
+            "path": "artist/Bulgarian-Online-Store-Feedback-Text-Analysis",
+            "revision": "701984d6c6efea0e14a1c7850ef70e464c5577c0",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2018-05-14", "2018-05-14"),
+            eval_splits=["test"],
+        eval_langs=["bul-Cyrl"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="cc-by-4.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""@data{DVN/TXIK9P_2018,
+author = {Georgieva-Trifonova, Tsvetanka and Stefanova, Milena and Kalchev, Stefan},
+publisher = {Harvard Dataverse},
+title = {{Dataset for ``Customer Feedback Text Analysis for Online Stores Reviews in Bulgarian''}},
+year = {2018},
+version = {V1},
+doi = {10.7910/DVN/TXIK9P},
+url = {https://doi.org/10.7910/DVN/TXIK9P}
+}
+""",
+        n_samples={"test": 182},
+        avg_character_length={"test": 316.7},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("Review", "text")
+        self.dataset = self.dataset.rename_column("Category", "label")
```

Comment:
label here should be of type `int` (see `class AbsTaskClassification`), where as this HF dataset's `Category` column is of type string. Each category needs to be mapped to an `int`.

Reply:
For class encoding, you can use `class_encode_columns()` from `datasets` (see docs [here](https://huggingface.co/docs/datasets/v2.18.0/en/package_reference/main_classes#datasets.Dataset.class_encode_column))

Reply:
Thanks for the example!